Add initial KVM 11.1.1 release notes (#89)
giantswarm/releases: release-notes/kvm/v11.1.0.md

## :zap: Giant Swarm Release 11.1.0 for KVM is now active for you! :zap:

> This release upgrades the NGINX Ingress Controller app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also added metrics Service for prometheus-operator support. This enables scraping of metrics and better monitoring on the app.

Reviewer: ```suggestion
This release upgrades the NGINX Ingress Controller app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also enabled the metrics service. This allows prometheus-operator to scrape for metrics and improves our monitoring of the app.
```

Same suggestion as in AWS release PR.

Author: > improves our monitoring of the app

Customer (adidas) requested the feature, for them to be able to monitor nginx IC, using prometheus-operator (managed prometheus); they manage that prometheus-operator too at the moment. We may eventually end up using this metrics Service too (enabling it by default), but not yet.

> This release upgrades the NGINX Ingress Controller app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also added metrics Service for prometheus-operator support. This enables scraping of metrics and better monitoring on the app.

Reviewer 2: ```suggestion
This release upgrades the NGINX Ingress Controller app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also enabled the metrics service. This allows prometheus-operator to scrape for metrics and improves monitoring of the app.
--

.
```

Author: @cokiengchiara will commit suggestion for most of the changes are improvement, with exception of "enabled" part -  we haven't actually enabled the metrics Service by default, we supported it, it has to be enabled on-demand.
